packages/orbit-components/src/hooks/useRandomId/index.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+import { useUIDSeed } from "react-uid";
+
+import type { useRandomID } from "./index.js.flow";
+
+const useRandomId: useRandomID = (name: string) => {
+  return useUIDSeed()(name);
```

Comment:
What's the benefit of `useUIDSeed` in this case over `useUID`? Considering that you're immediately calling the seed instead of saving it for using it multiple times.

Reply:
it's similar to previous behavior with argument. Not sure what you mean by `multiple times`, because for several id's the hook should be defined several times. Like: 

```
 const id1 = useUID();
 const id2 = useUID();
 const id3 = useUID();

```